src/ui/ScaleManager.js

```diff
@@ -36,8 +64,14 @@ var ScaleManager = exports = Class(function() {
 
 });
 
-ScaleManager.SCALE_MODE = {
-  NONE: "scaleModeNone",
-  LOCK_WIDTH: "scaleModeLockWidth",
-  LOCK_HEIGHT: "scaleModeLockHeight"
-};
\ No newline at end of file
+/**
+ * @var {object} ScaleManager.SCALE_MODE
+ * @property {string} NONE
+ * @property {string} LOCK_WIDTH
+ * @property {string} LOCK_HEIGHT
+ */
+exports.SCALE_MODE = {
```

Comment:
I don't think it's great to have a key and a value here, especially if they're both strings and different. It would be much easier and straight forward to just use the keys, i.e. `LOCK_WIDTH` and `LOCK_HEIGHT`